<launch>

  <!-- send pr2 urdf to param server -->
  <node pkg="tf" type="static_transform_publisher" name="link1_broadcaster" args="0 0 0 0 0 0 world_frae /base_footprint 10" />
  <param name="robot_description" command="$(find xacro)/xacro.py '$(find pr2_description)/robots/pr2.urdf.xacro'" />

  <!-- Planning Context -->
  <include file="$(find pr2_moveit_config)/launch/planning_context.launch" />

  <node pkg="benchmark_manipulation_tests" name="$(anon benchmark_manipulation_tests)" type="visualizePaths" output="screen" required="true">
    <rosparam command="load" file="$(find benchmark_manipulation_tests)/config/single_arm_countertop15.yaml" />
    <rosparam command="load" file="$(find benchmark_manipulation_tests)/config/locations_countertop.yaml" />
    <rosparam command="load" file="$(find benchmark_manipulation_tests)/config/experiments_countertop_first.yaml" />
    <param name="known_objects_filename" value="$(find benchmark_manipulation_tests)/env/countertop.env" />
    <rosparam command="load" file="$(find pr2_moveit_config)/config/kinematics.yaml"/>
    <param name="environment_name" value="countertop" />

    <param name="benchmark_results_folder" value="$(find benchmark_manipulation_tests)/results/" />


    <!-- Planners -->
    <rosparam command="load" file="$(find pr2_moveit_config)/config/kinematics.yaml"/>
    <rosparam command="load" file="$(find pr2_moveit_config)/config/ompl_planning.yaml"/>

    <rosparam command="load" file="$(find sbpl_arm_planner)/config/pr2_right_arm.yaml" />
    <param name="action_set_filename" value="$(find sbpl_arm_planner)/config/pr2.mprim" />
    <rosparam command="load" file="$(find sbpl_collision_checking)/config/pr2_right_arm_model.yaml" />

  </node>

</launch>
